Api/CustomStartup.cs

```diff
@@ -94,11 +94,10 @@ private static void SeedDatabaseIfEmpty(this WebApplication app)
         var ev2 = new Event(st1, "Ev2", "Des2", EventCategory.Uncategorized, week_ago, now, weeks_in, "Loc3", null, 10)
             { Guid = Guid.Parse("acbcd26e-877c-45cf-a697-ff4a3c70980a"), ViewCount = 2 };
         ev0.Participants.Add(st0); ev0.Participants.Add(st1);
-        ev0.Interested.Add(st0);
-        st0.SubscribedEvents.Add(ev0); st1.SubscribedEvents.Add(ev0);
+        st0.JoinedEvents.Add(ev0); st1.JoinedEvents.Add(ev0);
         ev1.Participants.Add(st0);
-        ev1.Interested.Add(st0); ev1.Interested.Add(st2);
-        st0.SubscribedEvents.Add(ev1); st2.SubscribedEvents.Add(ev1);
+        ev1.Interested.Add(st2);
+        st0.JoinedEvents.Add(ev1); st2.SubscribedEvents.Add(ev1);
         ev2.Participants.Add(st1);
```

Comment:
I think `st1.JoinedEvents.Add(ev2)` is missing here

Reply:
I forgot to remove ev2.Paticipants.Add(st2). Student cannot be both participant and interested